tests/test_main.py

```
@@ -1411,6 +1680,351 @@ async def mock_sync_forever(*args, **kwargs):
+                The active backend instance, or `None` if no backend is active.
+            """
+            return None
+
+        async def connect_active_backend(self, _config: dict[str, Any]) -> bool:
+            """
+            Stub implementation that signals this method must not be invoked.
+
+            Raises:
+                AssertionError: always raised with the message "connect_active_backend should not be called".
+            """
+            raise AssertionError("connect_active_backend should not be called")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)